src/debuginfo.cpp

```diff
@@ -343,7 +343,9 @@ class JuliaJITEventListener: public JITEventListener
         for (const object::SymbolRef &sym_iter : debugObj.symbols()) {
             StringRef sName;
 #ifdef LLVM37
-            sName = sym_iter.getName().get();
+            auto sNameOrError = sym_iter.getName();
+            assert(sNameOrError);
```

Comment:
Yes, the issue is that `Expected<T>::operator bool()` has to be called before invoking `Expected<T>::get()`, but only if LLVM was built with assertions enabled. So this would cause a problem only if you want to build Julia with assertions disabled together with LLVM where assertions are enabled, which I overlooked. But how can a release build of Julia be triggered? Because when I simply invoke the build process via `make` without any manual configuration, then Julia will still be built with assertions enabled. However, I think a release build of Julia would normally use a release build of LLVM anyway? Because then this should not cause any problems.